packages/client/src/components/Actions/components/ActionRequired.js

```diff
@@ -0,0 +1,65 @@
+import { ACTION_TYPES } from "constants/enums";
+import NFTView from "./NFTView";
+import TokenView from "./TokenView";
+import OwnerView from "./OwnerView";
+import ThresholdView from "./ThresholdView";
+import Confirmations from "./Confirmations";
+
+const ActionRequired = ({
+  safeData,
+  actionView,
+  confirmations,
+  onReject,
+  onApprove,
+}) => {
+  const { type: actionType } = actionView;
+
+  const getActionView = () => {
+    switch (actionType) {
+      case ACTION_TYPES.TRANSFER_TOKEN:
+        return <TokenView actionView={actionView} safeData={safeData} />;
+      case ACTION_TYPES.TRANSFER_NFT:
+        return <NFTView actionView={actionView} safeData={safeData} />;
+      case ACTION_TYPES.ADD_OWNER:
+        return (
+          <OwnerView actionView={actionView} safeData={safeData} isAdd={true} />
+        );
+      case ACTION_TYPES.REMOVE_OWNER:
+        return (
+          <OwnerView
+            actionView={actionView}
+            safeData={safeData}
+            isAdd={false}
+          />
+        );
+      case ACTION_TYPES.UPDATE_THRESHOLD:
+        return <ThresholdView actionView={actionView} safeData={safeData} />;
+      default:
+        return <></>;
```

Comment:
```suggestion
        return null
```
reason: no need to render an empty div